process-cmds.sh

```diff
@@ -202,98 +202,98 @@ inspect-system-ovs-ovn() {
   echo "### Command: ethtool -i ovn-k8s-mp0_0" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ethtool -i ovn-k8s-mp0_0\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: cat /proc/net/nf_conntrack" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /proc/net/nf_conntrack\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "" >> "${HWOL_VALIDATION_FILENAME}"
   echo "## OvS Information" >> "${HWOL_VALIDATION_FILENAME}"
   # Workaround: https://bugzilla.redhat.com/show_bug.cgi?id=1803920
   OPENVSWITCHPID=`kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"cat /var/run/openvswitch/ovs-vswitchd.pid\""`
   echo "### Command: ovs-vsctl show" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ovs-vsctl show\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ovs-vsctl show\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: sudo ovs-appctl dpctl/show" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/show\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/show\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "### Command: sudo ovs-appctl dpctl/dump-flows --names -m" >> "${HWOL_VALIDATION_FILENAME}"
-  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/dump-flows --names -m\"" >> "${HWOL_VALIDATION_FILENAME}"
+  kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"sudo ovs-appctl -t /var/run/openvswitch/ovs-vswitchd.${OPENVSWITCHPID}.ctl dpctl/dump-flows --names -m\" | cat" >> "${HWOL_VALIDATION_FILENAME}"
   echo "" >> "${HWOL_VALIDATION_FILENAME}"
   echo "## Kernel TC filter Information" >> "${HWOL_VALIDATION_FILENAME}"
   # Get PF from br-ex
   PF_NAME=`kubectl exec -n "${FT_NAMESPACE}" "${TEST_TOOLS_POD}" -- /bin/sh -c "chroot /host /bin/bash -c \"ovs-vsctl list-ports br-ex | grep -v patch-br-ex\""`
```

Comment:
Does this command work? Without the cat?

Reply:
it does not work without the cat. It does work with piping to cat.

Reply:
Great, that means we can keep it as is.